skglm/solvers/cd_solver.py

```diff
@@ -268,19 +285,27 @@ def cd_solver(
         for epoch in range(max_epochs):
```

Comment:
@mathurinm, I don't know whether it's worth it to check intercept optimality when solving subproblems. WDYT?